client/controller/index.web.js

```diff
@@ -26,7 +27,9 @@ import { render, hydrate } from './web-util.js';
 export { setSectionMiddleware, setLocaleMiddleware } from './shared.js';
 export { render, hydrate } from './web-util.js';
 
-const queryClient = new QueryClient();
+const queryClient = new QueryClient( {
+	queryCache,
```

Comment:
And used it in our query client.

Reply:
You don't need to provide your own `QueryCache` instance like this. The cache can be accessed through the `queryClient.getQueryData()` method.